maldoca/ole/ooxml_to_proto_cli.cc

```diff
@@ -23,7 +23,11 @@
 
 // Displays the proto message outputted for the file `filename`.
 // Note that `filename` contains the full path of the file.
-void DisplayFileContent(absl::string_view filename) {
+#ifndef MALDOCA_CHROME
+void DisplayFileContent(std::string filename) {
+#else
```

Comment:
with wrappers you just need to change absl::string_view to  const ::base::FilePath::StringType&